mock-kubernetes-client/client/client.go

```diff
@@ -89,7 +89,12 @@ func NewMockClientWithHooks(scheme *runtime.Scheme, createHooks []func(ctx conte
 			pod.Status.Phase = corev1.PodRunning
 		}
 
-		return nil
+		node := corev1.Node{
```

Comment:
Do you know what's the implication (test runtime) of adding the additional nodes is?